resources/util.sh

```diff
@@ -92,6 +92,18 @@ function configureNexusAtFirstStart() {
       doguctl template "${NEXUS_WORKDIR}/resources/nexusConfParameters.json.tpl" \
       "${NEXUS_WORKDIR}/resources/nexusConfParameters.json"
 
+    echo "Rendering cleanupPolicies template"
+    ADMINDEFAULTPASSWORD="${nexusPassword}" \
+      NEWADMINPASSWORD="${ADMINPW}" \
+      doguctl template "${NEXUS_WORKDIR}/resources/nexusCleanupPolicies.json.tpl" \
+      "${NEXUS_WORKDIR}/resources/nexusCleanupPolicies.json"
+
+    echo "Rendering compactBlobstore template"
+    ADMINDEFAULTPASSWORD="${nexusPassword}" \
+      NEWADMINPASSWORD="${ADMINPW}" \
```

Comment:
Where will these variables be used? The templates itself don't need them.

Reply:
See above